ONSdigital/ras-secure-message
# Batch respondent and business calls

**What is the context of this PR?**

https://trello.com/c/cJMHZnbR/216-update-secure-message-to-use-new-party-service-batch-call-5

The purpose of this PR is to update secure messaging to use the batch call in the party service repo for the businesses and respondents.

**How to review**

- Run make test and all tests should pass

- Run the acceptance tests to get some messages and you should be able to see all messages on R-ops
